LICENSE

```diff
@@ -1,4 +1,16 @@
-gTile is being released under GPL v2+ license
+Copyright (C) 2023  Markus Schanz
```

Comment:
This doesn't seem right.

Reply:
How so? Check https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html

Reply:
I'll resolve this. If there are legal issues that would argue against having my name in the LICENSE file, please provide an explanation, best accompanied with a link of an authoritative entity.